src/frontend/pages/Notes/Notes.jsx

```diff
@@ -15,10 +15,13 @@ import { useAuth } from "../../context/auth-context";
 import { useNotes } from "../../context/notes-context";
 import { addToArchives } from "../../services/notes-archive-service";
 import { addToTrash } from "../../services/notes-trash-service";
```

Comment:
suggestion: barrel export can be used

Reply:
Thanks for the suggestion, this will be done in a future PR with refactored code